users/admin.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib import admin
+
+from .models import User
+
+admin.site.register(User)
```

Comment:
override the admin model of the built-in user. I cannot simply change my password in the admin with your changes